WISE-Community/WISE
# Styled student add project dialog

Closes #1545.

To test:
- Log in as a student.
- Make sure adding a new unit (run) works.
- Try with an invalid access code and an access code for a run you've already added. You should see error messages for both cases.
- Make sure the 'Add' button is disabled until both fields are valid.
